polling_stations/apps/uk_geo_utils/helpers.py

```diff
@@ -19,6 +19,8 @@ def get_onsud_model():
 class Postcode:
 
     def __init__(self, postcode):
+        if len(str(postcode)) < 5:
```

Comment:
Shouldn't this be _after_ stripping spaces and other non-alphanumerics?

Reply:
Yes - good spot